vtm-desktop/src/org/oscim/gdx/GdxMapApp.java

```diff
@@ -62,6 +63,14 @@ protected static LwjglApplicationConfiguration getConfig(String title) {
         LwjglApplicationConfiguration.disableAudio = true;
         LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
         cfg.title = title != null ? title : "vtm-gdx";
+
+        String[] imgDimensions = new String[]{"16", "32", "64", "128"};
+        for (String imgDimension : imgDimensions) {
+            String path = "assets/launcher/ic_vtm_" + imgDimension + ".png";
+            //if (Gdx.files.internal(path).exists()) // Check if file exist!
```

Comment:
I'm not sure if icons are detected in any system/environment. So I wanted to check if file exists, but this isn't possible with gdx tools as they aren't initialized at this moment.